docs/guides/cli/15_config-identity-service.mdx

```diff
@@ -0,0 +1,77 @@
+---
+id: identity-cli
+title: Configure Identity Service
+---
+
+Using the Ory CLI, you can quickly adjust the configuration of the Ory Cloud
+Identity Service and apply config from the Cloud environment to self-hosted
+Ory Kratos instances. 
+
+:::tip
+
+Ory Kratos is the open-source project that powers the Ory Cloud Identity Service.
+
+:::
+
+This feature is useful for fluently moving your local setup to the Cloud, or 
+for working on the configuration and testing different settings in the safety 
+of the local development environment. 
+
+## Export Configuration from Ory Cloud
+
+To get your project's Identity Service configuration, run this command:
+
+:::info
+
+You must be signed in to an Ory Cloud account to perform this action. 
+Read [this document](./cli-basics#use-cli-with-existing-cloud-account) to learn more. 
+
+:::
+
+```bash
+ory get identity-config <project-id> --format yaml
+```
+
+The `--format` flag defines the format of the output file. You can choose one of `yaml`
+,`json`, or `json-pretty`. 
+
+Getting the configuration in the YAML format produces output that is ready 
+to use with self-hosted Ory Kratos. 
+
+### Export to File
+
+For a convenient way to get the Identity Service configuration 
+working in self-hosted Ory Kratos, save the configuration to a YAML file.
+
+Run this command:
+
+```bash
+ory get identity-config <project-id> --format yaml >identity-config.yaml 
+```
+
+## Import Configuration to Ory Cloud
+
```

Comment:
it would be nice to add a hint with creating a project as an entry to this page itself

Reply:
I added one that links to the CLI docs and the console. I didn't put in a "tip" or "note" frame though as there are already a few of those in the document, especially in its first part.